src/components/AddButton.spec.js

```diff
@@ -0,0 +1,22 @@
+import AddButton from './AddButton';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('AddButton', () => {
+  it('renders a button', () => {
+    const onClick = jest.fn();
+    render(<AddButton onClick={onClick} />);
+
+    const button = screen.getByRole('button');
```

Comment:
Maybe add the `{ name: '...' }` to be even more specific?

Reply:
I tried to test it this way, but i have to write in my test:
`{name: "plus_icon.svg zur Watchlist hinzufügen"}` 
Would this be okay? Because there is the icon and a screenreaderonly text? 🤔